Composer/packages/client/src/components/GetStarted/GetStarted.tsx

```diff
@@ -4,148 +4,65 @@
-              </Link>
-            </li>
-            <li style={liStyle}>
-              <Link href={linkToLUEditor} onClick={linkClick}>
-                {formatMessage('Train your language model')}
-              </Link>
-            </li>
-            <li style={liStyle}>
-              <Link href={linkToConnections} onClick={linkClick}>
-                {formatMessage('Connect your bot to new services')}
-              </Link>
-            </li>
-          </ul>
-          {formatMessage('Publish')}
-          <ul style={ulStyle}>
-            <li style={liStyle}>
-              <Link href={linkToProvision} onClick={linkClick}>
-                {formatMessage('Create a cloud hosting environment')}
-              </Link>
-            </li>
-            <li style={liStyle}>
-              <Link href={linkToPublish} onClick={linkClick}>
-                {formatMessage('Publish updates to the cloud')}
-              </Link>
-            </li>
-          </ul>
-          <h3 style={h3Style}>{formatMessage('Guides and references')}</h3>
-          <ul style={ulStyle}>
-            <li style={liStyle}>
-              <Link href={linkToGetStarted} target="_blank" onClick={linkClick}>
-                {formatMessage('Get started with Bot Framework Composer')}
-              </Link>
-            </li>
-            <li style={liStyle}>
-              <Link href={linkToCreateFirstBot} target="_blank" onClick={linkClick}>
-                {formatMessage('Create your first bot')}
-              </Link>
-            </li>
-            <li style={liStyle}>
-              <Link href={linkToTutorials} target="_blank" onClick={linkClick}>
-                {formatMessage('Composer tutorials')}
-              </Link>
-            </li>
-            <li style={liStyle}>
-              <Link href={linkToAdaptiveExpressions} target="_blank" onClick={linkClick}>
-                {formatMessage('Learn about Adaptive expressions')}
-              </Link>
-            </li>
-            <li style={liStyle}>
-              <Link href={linkToPreBuiltExpressions} target="_blank" onClick={linkClick}>
-                {formatMessage('Find pre-built Adaptive expressions')}
-              </Link>
-            </li>
-            <li style={liStyle}>
-              <Link href={linkToLUFileFormat} target="_blank" onClick={linkClick}>
-                {formatMessage('LU file format and syntax')}
-              </Link>
-            </li>
-            <li style={liStyle}>
-              <Link href={linkToLGFileFormat} target="_blank" onClick={linkClick}>
-                {formatMessage('LG file format and syntax')}
-              </Link>
-            </li>
-          </ul>
-        </Stack.Item>
-      </Stack>
-    </Panel>
+    <Panel
+      isBlocking={false}
+      isOpen={props.isOpen}
+      styles={{
+        root: {
+          marginTop: '50px',
+        },
+        navigation: {
+          display: 'block',
+          height: 'auto',
+        },
+      }}
```

Comment:
Nit: Putting the styles in a const avoid creating a new structure every render.  It can be a pain to get the typing to match up sometimes with Fluent IBlahStyles type.